invoice-service/src/main/java/com/registry/invoiceservice/entity/Invoice.java

```diff
@@ -11,11 +11,12 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @Data
+@Table(name = "invoice")
```

Comment:
I think you don't need the @Table annotation if name attribute has the same value as the name of your entity. Speaking of entities, I don't see an @Entity annotation here.